hole-punch-interop/rust-relay/Dockerfile

```diff
@@ -1,18 +1,33 @@
 # syntax=docker/dockerfile:1.5-labs
-FROM rust:1.72.0 as builder
 
-RUN rustup target add x86_64-unknown-linux-musl
-RUN --mount=type=cache,target=/var/cache/apt apt-get update && apt-get install -y musl-dev musl-tools
+FROM --platform=$BUILDPLATFORM rust:1.72.0 as builder
+
+ARG BUILDPLATFORM
+
+# Copy the setup.sh script into the image
+COPY setRustTarget.sh /setRustTarget.sh
```

Comment:
What do you think of just calling this `build.sh` and doing the entire build within the shell-script? Then we don't need to export the variables anywhere.

Reply:
It's a great idea, thanks.